tests/test_errorcallback.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for the error callback"""
+
+import unittest
+import unittest.mock
```

Comment:
I looked at unittest, the library already in use, searched on the page for a way to test for function calls, saw unittest.mock, and used that. So, Python 2's version of unittest doesn't have .mock?

The import switch doesn't work for me when I run "python2 setup.py test": 'No module named mock'

As for pytest, is that a pypi package? How would one go about adding it to the project's dependencies. Or should one?

I also get 'No module named osc2midi.oscdispatcher'